src/org/zaproxy/zap/GuiBootstrap.java

```diff
@@ -345,15 +340,37 @@ private static void setDefaultViewLocale(Locale locale) {
      * {@link java.net.InetAddress InetAddress}) preventing some ZAP options from being correctly applied.
      */
     private void setupLookAndFeel() {
-        if (lookAndFeelSet) {
-            return;
+    	
+    	String lookAndFeelClassnameFromCommandline = System.getProperty("swing.defaultlaf");
+    	OptionsParam options = Model.getSingleton().getOptionsParam();
+    	String lookAndFeelnameFromOptions = options.getViewParam().getLookAndFeel();
+    	String lookAndFeelClassnameFromOptions = "";
+    	UIManager.LookAndFeelInfo[] looks = UIManager.getInstalledLookAndFeels();
+    	
+    	for(UIManager.LookAndFeelInfo index : looks) {
+    		if(index.getName().equals(lookAndFeelnameFromOptions)) {
+    			lookAndFeelClassnameFromOptions = index.getClassName();
+    			break;
+    		}
+    	}
+    	
+       if ( lookAndFeelClassnameFromCommandline!= null) {
+        	try{
```

Comment:
It should be extracted a method that sets the L&F (doing exactly the same in both cases).